Client/Application/UITestAppDelegate.swift

```diff
@@ -60,7 +60,11 @@ class UITestAppDelegate: AppDelegate, FeatureFlaggable {
                     try? FileManager.default.removeItem(at: URL(fileURLWithPath: "\(dirForTestProfile)/\(item)"))
                 }
 
-                try! FileManager.default.copyItem(at: input, to: output)
+                do {
```

Comment:
Not sure about these changes here, some tests that have been really stable are failing in this PR. Please let me take a look at this PR before landing

Reply:
Looks like the launch arguments are not working as expected in some scenarios with these changes and so the tests fail... 
For example, for this test: testFirstRunTour the Onboarding is not shown but it should be there since we have a setUp method with specific arguments.

Reply:
thank you @isabelrios I will take a look into the failure I feel it can be related to one of the unwrapping changes and it could be failing to load the onboarding flow

Reply:
thanks @yoanarios let me know if you need more details about the failures

Reply:
I checked the onboarding test and I can reproduce the failure locally.
Note: The test are passing on main and when running the app the onboarding is shown properly so the failure seems related to the test scheme only. I couldn't fin any evident cause at first glance

Reply:
The issue is that the onboarding should be shown in the test but it is not. We skip the onboarding in all tests with a specific launch argument. In this case we are overwriting the set up so that the onboarding is shown in all the tests in this tes suite. The launch arguments are managed in this file, that’s why my feeling is that somehow changing this force try are breaking something with the way the app is launched for testing

Reply:
One way we could test is by disabling the `force_try` rule for the lines in the file and revert the rest of these changes.  I'm not sure how force try would change the launch arguments unless there is some recovery mechanism in the test plans.  But we can see if you're right with such a change

Reply:
I will be doing some changes to try to isolate the issue. Thanks for your help